cupyx/scipy/sparse/linalg/_norm.py

```diff
@@ -0,0 +1,167 @@
+import numpy
+
+import cupy
+from cupy.cuda import device
+import cupyx.scipy.sparse
+
+__all__ = ['norm']
+
+
+def _sparse_frobenius_norm(x):
+    if cupy.issubdtype(x.dtype, cupy.complexfloating):
+        sqnorm = abs(x).power(2).sum()
+    else:
+        sqnorm = x.power(2).sum()
+    return cupy.sqrt(sqnorm).item()
+
+def norm(x, ord=None, axis=None):
+    """
+    Norm of a cupy.scipy.spmatrix
+    This function is able to return one of seven different matrix norms,
+    depending on the value of the ``ord`` parameter.
+    Parameters
+    ----------
+    x : a sparse matrix
+        Input sparse matrix.
+    ord : {non-zero int, inf, -inf, 'fro'}, optional
+        Order of the norm (see table under ``Notes``). inf means numpy's
+        `inf` object.
+    axis : {int, 2-tuple of ints, None}, optional
+        If `axis` is an integer, it specifies the axis of `x` along which to
+        compute the vector norms.  If `axis` is a 2-tuple, it specifies the
+        axes that hold 2-D matrices, and the matrix norms of these matrices
+        are computed.  If `axis` is None then either a vector norm (when `x`
+        is 1-D) or a matrix norm (when `x` is 2-D) is returned.
+    Returns
+    -------
+    n : float or ndarray
+    Notes
+    -----
+    Some of the ord are not implemented because some associated functions like, 
+    _multi_svd_norm, are not yet available for sparse matrix. 
+    This docstring is modified based on numpy.linalg.norm. 
+    https://github.com/numpy/numpy/blob/master/numpy/linalg/linalg.py 
```

Comment:
- Empty lines are required before section headers (in NumPy Style).
- (Optional in this PR) Follow Google Style (see e.g. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) like most of the other docstrings in CuPy.

Reply:
Please Review